scripts/nb-tester/notebooks.toml

```diff
@@ -0,0 +1,20 @@
+# Used to find all notebooks in the repo
+all-notebooks = "[!.]*/**/*.ipynb"
+
+# Always exclude notebooks matching the following patterns
+notebooks-exclude = [
+    "docs/api/**",
```

Comment:
These shouldn't exist for actual APIs. This would also unintentionally ignore `/api/migration-guides`. 

```suggestion
```

You could make this the default option so that it doesn't have to be set at all in TOML. I imagine it will be the same value for closed source?